_inc/client/components/settings-card/index.jsx

```diff
@@ -10,42 +10,58 @@ import { translate as __ } from 'i18n-calypso';
 import SectionHeader from 'components/section-header';
 import Button from 'components/button';
 
-const SettingsCard = props => {
```

Comment:
As discussed in Slack, this should be restored as a stateless component since no component state methods provided by React.createClass are used.